### PR TITLE
Reformat General - value_is_nan #1885 #1928

### PR DESCRIPTION
**Pull request by brgopalakrishnan**
Added Functional Examples to value_is_nan. And based on the comments in #1928 

"The PR looks quite messy as it isn't clear what changes were made by you. You could fetch all the changes and only push yours but I think it is easier to just create another PR so I will be closing this one, please mention the issue you were working on in the new PR as well. Thanks!"

Redid the PR.

### CODE REVIEW

**Comment by brgopalakrishnan** [comment]
Working on fixing lint issues

**Comment by Darshan-H-E** [comment]
@MarShaikh, please review this PR.

**Review by MarShaikh — CHANGES_REQUESTED** [review]
Thanks for the comprehensive examples! They look great! Could you please remove the instance method examples as they don't belong here.

**Comment by brgopalakrishnan** [comment]
> Thanks for the comprehensive examples! They look great! Could you please remove the instance method examples as they don't belong here.

Thank you for Reviewing. Just pushed the requested changes.

**Review by MarShaikh — CHANGES_REQUESTED** [review]
Thanks for the update. They look great! Sorry for being a bit nit picky, but you can see that some examples are redundant, I'm sure you can remove a few, especially from the container, and native array sections. Just make sure to re-request for a review!

**Comment by brgopalakrishnan** [comment]
> Thanks for the update. They look great! Sorry for being a bit nit picky, but you can see that some examples are redundant, I'm sure you can remove a few, especially from the container, and native array sections. Just make sure to re-request for a review!

no worries, removed the redundant as far as I can see. please review and let me know your thoughts.

File: ivy/functional/ivy/general.py [hunk 1]
```diff
@@ -1456,6 +1456,184 @@ def value_is_nan(
...
+    --------
+
+    With :code:`ivy.Array` input:
+
+    >>> x = ivy.array([1])
+    >>> y = ivy.value_is_nan(x)
+    >>> print(y)
+    False
+
+    >>> x = ivy.array([1])
```
Comment (MarShaikh) [inline ivy/functional/ivy/general.py:1469]: Just to make things neater, you can remove this example and the one after this.

File: ivy/functional/ivy/general.py [hunk 2]
```diff
@@ -1456,6 +1456,184 @@ def value_is_nan(
...
+    False
+
+    With :code:`ivy.NativeArray` input:
+
+    >>> x = ivy.native_array([1])
+    >>> y = ivy.value_is_nan(x)
+    >>> print(y)
+    False
+
+    >>> x = ivy.native_array([1])
```
Comment (MarShaikh) [inline ivy/functional/ivy/general.py:1491]: Similarly you can remove the other 3 examples from here as well!

File: ivy/functional/ivy/general.py [hunk 3]
```diff
@@ -1456,6 +1456,184 @@ def value_is_nan(
...
+    False
+
+    With :code:`Number` inputs:
+
+    >>> x = 1
+    >>> y = ivy.value_is_nan(x)
+    >>> print(y)
+    False
+
+    >>> x = 1
```
Comment (MarShaikh) [inline ivy/functional/ivy/general.py:1513]: Same here too!

File: ivy/functional/ivy/general.py [hunk 4]
```diff
@@ -1456,6 +1456,184 @@ def value_is_nan(
...
+    }
+
+    >>> x = ivy.Container(a=ivy.array([3]))
+    >>> y = ivy.value_is_nan(x, False)
+    >>> print(y)
+    {
+        a: false
+    }
+    
+    >>> x = ivy.Container(a=ivy.array([3]), \
```
Comment (MarShaikh) [inline ivy/functional/ivy/general.py:1544]: Finally, this one example for container is enough. 🙂

**Review by MarShaikh — CHANGES_REQUESTED** [review]
Just added few more comments. You didn't need to create a grid search of examples, but they were great nonetheless! I've only asked you to remove those as it helps to keep the docstrings of these functions fairly neater. Thanks, I'll be able to merge after you make these changes.

File: ivy/functional/ivy/general.py (hunk 1 above)
Comment (brgopalakrishnan) [reply ivy/functional/ivy/general.py:1482]: Thank you for your comments. Removed .

File: ivy/functional/ivy/general.py (hunk 2 above)
Comment (brgopalakrishnan) [reply ivy/functional/ivy/general.py:1504]: Thank you for your comments. Removed .

File: ivy/functional/ivy/general.py (hunk 3 above)
Comment (brgopalakrishnan) [reply ivy/functional/ivy/general.py:1526]: Thank you for your comments. Removed .

File: ivy/functional/ivy/general.py (hunk 4 above)
Comment (brgopalakrishnan) [reply ivy/functional/ivy/general.py:1557]: Thank you for your comments. Removed .

**Comment by brgopalakrishnan** [comment]
@MarShaikh Removed as per the requirement, whenever you get a chance let me know how can I help to resolve it to get into master.

**Review by MarShaikh — CHANGES_REQUESTED** [review]
It seems as though you have only added those changes to your local repo. You can update this PR by simply pushing to your main branch from local. Request for a re-review after you're done. Sorry for the delay in reviewing!

**Pull request closed without merging**